### PR TITLE
Georgian (GE) Translation

### PR DESCRIPTION
**Pull request by wikicrafter**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

I have followed the guideline and  I'll  make full translation in Georgian (ge) 
language.

### CODE REVIEW

**Comment by christ1980** [comment]
> * [ ]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [ ]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 
> I have followed the guideline and soon I'll make full translation

**Review by christ1980 — COMMENTED** [review]
Ok

**Review by christ1980 — COMMENTED** [review]
Ok

**Comment by wikicrafter** [comment]
Why script/test isn't working

> Ok

I 

> Ok



> Ok



I don't know why test is not working 
When I run test It shows next error : 


``` \opensource.guide\test\prose
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (C:\Env\opensource.guide\test\prose:4:15)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'C:\\Env\\opensource.guide\\test\\prose' ]
```

| OS | Version | 
| ------------- |:-------------:|
| Windows    | 11    |

**Comment by wikicrafter** [comment]
> Ok



> > * [x]   Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> > * [ ]   Have you explained what your changes do, and why they add value to the Guides?
> > 
> > **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> > I have followed the guideline and soon I'll make full translation



> Ok

Why script/test isn't working

> Ok




I don't know why test is not working 
When I run test It shows next error : 


``` \opensource.guide\test\prose
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (C:\Env\opensource.guide\test\prose:4:15)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ 'C:\\Env\\opensource.guide\\test\\prose' ]
```

| OS | Version | 
| ------------- |:-------------:|
| Windows    | 11    |

**Review by Klp2121 — COMMENTED** [review]
Lol

**Comment by chau2313** [comment]
>     * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> 
>     * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 
> I have followed the guideline and I'll make full translation in Georgian (ge) language.

File: Gemfile.lock [hunk 1]
```diff
@@ -25,6 +25,7 @@ GEM
     ethon (0.15.0)
       ffi (>= 1.15.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```
Comment (MikeMcQuaid) [inline Gemfile.lock:28]: Please remove these changes, thanks!

Comment (wikicrafter) [reply Gemfile.lock:28]: @MikeMcQuaid  All of them or just 28th line ```eventmachine (1.2.7-x64-mingw32) ```

Comment (MikeMcQuaid) [reply Gemfile.lock]: All `Gemfile` related changes, thanks!

Comment (wikicrafter) [reply Gemfile.lock]: Thanks for assisting!
 it shows same error when I type: ```script/test```

**Review by yara7533 — COMMENTED** [review]
#2606

**Review by jacobtrujillo0610 — COMMENTED** [review]
I’m trying not to be rude or anything but I’m just not sure what to expect from my comments or anything

**Review by Nab985 — COMMENTED** [review]
> Tyh

**Review by alyonatishchenko — COMMENTED** [review]
Ok..good))

**Comment by Antonio3m** [comment]
Bien

**Review by ivanpoh99 — COMMENTED** [review]
i

**Comment by anilk666** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**
> 
> I have followed the guideline and I'll make full translation in Georgian (ge) language.

Yes

**Comment by mohammadhosein552zarei** [comment]
bc1qahrsfmm82hqmwwwn6yqdtvw4nu474fnwn6ed5s

**Review by Fii02kong — COMMENTED** [review]
jhhhjjh

**Pull request closed without merging**